genesyscloud/resource_genesyscloud_outbound_attemptlimit.go

```diff
@@ -0,0 +1,334 @@
+package genesyscloud
```

Comment:
You'll have to add the resource exporter stuff. I haven't added code generation for that yet